### DISCUSSION TITLE
Run in webgl? New to using Vroid SDK! Need some Guideline!

### DISCUSSION THREAD

**Discussion by Nelsonlala** [pixiv/vroid-sdk-developers#44] [Q&A]
Hi, 

I just downloaded the VRoid SDK, And I would like to integrate it into my WebGL multiplayer application, used as player's avatars. 

But I have a few questions,  
1) Is Void SDK can run for the WebGL platform?
2) since I am making a multiplayer application, do I need to register for the multiplayer SDK ( and what's the difference?) 
3) is there a step to step eng guideline to implement the void SDK or any tutorial video that can follow, it would be great to have that to start as I feel very lost now.
4) is there a discord channel for us the developer or just here?

Thank you for your attention. 

Thanks,
Nelson

**Comment by nyamadan** [comment]
Hi.

> Is Void SDK can run for the WebGL platform?

No, VRoid SDK does not support WebGL builds.

> since I am making a multiplayer application, do I need to register for the multiplayer SDK ( and what's the difference?)

In addition to the regular SDK, you will have to apply for the multiplayer SDK. https://github.com/pixiv/vroid-sdk-developers/discussions/37
However, we do not provide a multiplayer SDK for WebGL.

> is there a step to step eng guideline to implement the void SDK or any tutorial video that can follow, it would be great to have that to start as I feel very lost now.

No video exists.
Please see the README included with the unitypackage for implementation instructions.

> is there a discord channel for us the developer or just here?

We do not have an official Discord Channel for developers.
Please use GitHub Discussion. https://github.com/pixiv/vroid-sdk-developers/discussions/2

[nyamadan](https://github.com/pixiv/vroid-sdk-developers/discussions/3#discussioncomment-1877844)
